_install-and-configure/configuring-opensearch/network-settings.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Network settings
+parent: Configuring OpenSearch
+nav_order: 20
+---
+
+# Network settings
```

Comment:
Would it make sense to include some explanatory content about what "HTTP communication" and "transport communication" is in this context? "communication between nodes" is mentioned lower down in one of the transport settings, but I'm thinking it might be helpful to have a sentence or two that explains that the HTTP settings are used to configure communication with external clients via the REST API and transport settings are for internal node-to-node communication within OpenSearch.

Reply:
Thank you, @andrross! I implemented your comments.